airflow/providers/microsoft/azure/hooks/msgraph.py

```diff
@@ -190,7 +190,7 @@ def get_conn(self) -> RequestAdapter:
             client_id = connection.login
             client_secret = connection.password
             config = connection.extra_dejson if connection.extra else {}
-            tenant_id = config.get("tenant_id")
+            tenant_id = config.get("tenantId") or config.get("tenant_id")
```

Comment:
If we want to change this then we should deprecate the old one.
But also, lets verify all settings. We should not mix two types of syntax's.

Reply:
@w0ut0 thanks for your PR. I would do following:

tenant_id = config.get(“tenant_id”) or config.get(“tenantId”)

No need to check the connection type. I would like to also use the Azure connection type in the future for the MSGraphAsyncOperator as that would make more sense but then some changes would not to be done on the Azure connection type. This would also be interesting for the PR regarding the PowerBi dataset operator of @ambika-garg. But as I said that would require some changes in the Azure connection type, something I didn’t want to mess with yet when I introduced the MSGraphOperator.